.github/workflows/publish-cli-bins.yaml

```
@@ -9,89 +9,159 @@ on:
-      - name: "Verify"
+      - name: Setup QEMU
         run: |
-          chmod +x flightctl-${{ matrix.env.build }}
-          ./flightctl-${{ matrix.env.build }} version
-          exit $?
+          if [[ "${{ matrix.os_arch }}" == "linux-arm64" ]]; then
+            docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          fi
+
+      - name: Verify
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2181:style:7:6: Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?

(shellcheck)